src/services/worker/backupUtils.ts

```
@@ -1271,6 +1337,10 @@ const dbRestoreSchedules = async (
         (record) => record.weekOf === remoteItem.weekOf
       );
 
+      if (typeof remoteItem.midweek_meeting.aux_fsg === 'string') {
+        delete remoteItem.midweek_meeting.aux_fsg;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)